src/interactivity/onecore/BgfxEngine.cpp

```diff
@@ -230,15 +230,20 @@ BgfxEngine::BgfxEngine(PVOID SharedViewBase, LONG DisplayHeight, LONG DisplayWid
     return S_OK;
 }
 
-std::vector<til::rectangle> BgfxEngine::GetDirtyArea()
+[[nodiscard]] HRESULT BgfxEngine::GetDirtyArea(gsl::span<const til::rectangle>& area) noexcept
 {
     SMALL_RECT r;
     r.Bottom = _displayHeight > 0 ? (SHORT)(_displayHeight - 1) : 0;
     r.Top = 0;
     r.Left = 0;
     r.Right = _displayWidth > 0 ? (SHORT)(_displayWidth - 1) : 0;
 
-    return { r };
+    _dirtyArea = r;
+
+    _area = { &_dirtyArea,

```

Comment:
I still still don't understand why this compiles. o_O

Reply:
cuz I fixed it against the razzle build and then failboated at carrying it back to the openconsole one. oops.